docs/force-move/consensus-game.md

```diff
@@ -0,0 +1,41 @@
+---
+id: consensus-game
+title: The Consensus Game
+---
+
+Purpose: advance the channel outcome only if all participants are in agreement.
+
+The application data has the format:
+
+```solidity
+struct ConsensusGameData {
+  uint32 furtherVotesRequired;
+  bytes currentOutcome;
```

Comment:
I think we can omit this here an pull it out of the `VariablePart` of the state.